python_cross_site_scripting/QGB_QPSU/HTML.py

```
@@ -0,0 +1,2126 @@
+	response.set_data(html)
+	return 	html
+fsimg=imgfs=fullscreen_img
+
+def a_href(response,u=''):
+	if not u:
+		u=U.cbg()
+	u=N.auto_url(u)
+	html='<br>'*5+f"""
+<a href="{u}">{u}</a>	
+	"""
+	response.headers['Content-Type']='text/html;charset=utf-8';
```

Ruff (0.13.3)

Statement ends with an unnecessary semicolon

Remove unnecessary semicolon

(E703)